llvm/utils/lit/tests/Inputs/shtest-cat/cat.txt

```diff
@@ -0,0 +1,83 @@
+# Check cat command with single file.
+#
+# RUN: rm -rf %T/testCat
```

Comment:
I know these are the existing test lines, but we’ve deprecated %T. Instead use %t. I think you also probably do need the testCat subdirectory either.

Reply:
I'd normally agree with you, but as the intent is to make various modifications to improve these existing tests in a follow-up, I think it makes sense to defer this change to that patch too, to make it clearer what actually changed.

I'm ambivalent about the extra directory. Happy either way on that one.

Reply:
I can make these changes in my follow-up patch. For changing %T to %t, how would I get the filepath of %t? I tried looking through the lit documentation and existing tests and I couldn't find a clear answer. Is it something like %t/dir?

Reply:
Yes that does - so %t is just a more versatile option that encapsulates %T, which is why %T is not used anymore?

Reply:
As I understand it, the main problem with `%T` is that it isn't unique to the individual test, so you could end up with two tests trying to create `%T/foo.o` and you'd end up with race conditions if the two were run in parallel. The same goes for directories. It may be possible to avoid these name clashes, but in practice, doing so takes more mental power than is really necessary.